File: game_server/logger.go
package main

import (
	"fmt"
	"time"
)

//print warning to stdout
func Warning(format string, v ...interface{}) {
	message := fmt.Sprintf(format, v...)
	fmt.Println("\033[93mWARNING : " +
		time.Now().UTC().Format(time.RFC3339) + " : " + message + "\033[0m")
}

//print error to stdout
func Error(format string, v ...interface{}) {
	message := fmt.Sprintf(format, v...)
	fmt.Println("\033[91mERROR : " +
		time.Now().UTC().Format(time.RFC3339) + " : " + message + "\033[0m")
}

//print info to stdout
func Info(format string, v ...interface{}) {
	message := fmt.Sprintf(format, v...)
	fmt.Println("\033[94mINFO : " +
		time.Now().UTC().Format(time.RFC3339) + " : " + message + "\033[0m")
}

File: client/ahriman/logger.py
def warning(*print_args, title=""):
    print('\033[93m\033[1mWarning\033[0m\033[93m {} : \033[0m'.format(title), *print_args)


def error(*print_args, title=""):
    print('\033[91m\033[1mError\033[0m\033[91m {} : \033[0m'.format(title), *print_args)


def info(*print_args, title=""):
    print('\033[94m\033[1mInfo\033[0m\033[94m {} : \033[0m'.format(title), *print_args)


def confirm(*print_args, title=""):
    print('\033[32m\033[1mSuccess\033[0m\033[32m {} : \033[0m'.format(title), *print_args)
